PluginGenerator/PluginBuilder.cs

```diff
@@ -196,46 +198,89 @@ public void Build()
                 throw new InvalidOperationException(UIResources.JarB_JDK_NotInstalled);
             }
 
-            this.Validate();
-
             // Temp working folder
-            string tempWorkingDir = Path.Combine(Path.GetTempPath(), "plugins",  Guid.NewGuid().ToString());
+            string tempWorkingDir = Utilities.CreateTempDirectory(".builder");
```

Comment:
Changed the root working folder so it will appear under "%temp%\.sqsdk" with the other temp files.